scripts/HostButton.gd

```diff
@@ -0,0 +1,17 @@
+extends Node
+
+signal set_connect_type
```

Comment:
The signal triggers the connecting splash screen, rather than setting if it's the host. It sense to separate the setting up of the server/client, and the visuals that get shown to the user. Though, the wording I've used certainly makes this unclear.